plugin/storage/integration/es_index_rollover_test.go

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build index_rollover
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strconv"
+	"testing"
+
+	"github.com/olivere/elastic"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	indexILMName  = "jaeger-ilm-policy"
+	rolloverImage = "jaegertracing/jaeger-es-rollover:latest"
+)
+
+func TestIndexRollover_FailIfILMNotPresent(t *testing.T) {
+	client, err := createESClient()
+	require.NoError(t, err)
+	esVersion, err := getVersion(client)
+	require.NoError(t, err)
+	if esVersion != 7 {
+		t.Skip("Integration test - " + t.Name() + " against ElasticSearch skipped for ES version " + fmt.Sprint(esVersion))
+	}
+	// make sure ES is clean
+	cleanES(t, client, "jaeger-ilm-policy")
+	envVars := []string{"ES_USE_ILM=true"}
+	errorMsg := "exit status 1"
```

Comment:
nit: can be inlined: `assert.EqualError(t, err, "exit status 1")`

Reply:
Fixed.